en/intro_to_command_line/README.md

```diff
@@ -72,7 +72,7 @@ Each command will be prepended by this sign and one space, but you don't have to
 
 The part up to and including the `$` or the `>` is called the *command line prompt*, or *prompt* for short. It prompts you to input something there.
 
-In the tutorial, when we want you to type in a command, we will include the `$` or `>`, and occasionally more to the left. You can ignore the left part and just type in the command which starts after the prompt.
+In the tutorial, when we want you to type in a command, we will include the `$` or `>`, and occasionally more to the left. You can ignore the left part and type in the command which starts after the prompt.
```

Comment:
sure.

Reply:
I'll see if I can figure out how to update the PR for this line.

Reply:
Actually I have no idea how to update a PR on Github. Any hints?

Reply:
I think I figured it out. Committed another change to this line.

Reply:
> Actually I have no idea how to update a PR on Github. Any hints?

In case someone else wonders about this: <del>just</del> make a new commit to the branch your PR requests to be merged (i.e. your own branch).

If you made that commit on your local computer, you'll <del>obviously</del> have to push it to GitHub, so that the branch on GitHub is updated, too. If you make the commit by using the GitHub web editor to change a file on your branch, the commit will <del>of course</del> already be on the right branch.

As long as the PR hasn't been merged yet, all new commit on the corresponding GitHub branch will automatically become part of the PR.